mmseg/models/decode_heads/segformer_head.py

```diff
@@ -0,0 +1,60 @@
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule
+
+from mmseg.models.builder import HEADS
+from mmseg.models.decode_heads.decode_head import BaseDecodeHead
+from mmseg.ops import resize
+
+
+@HEADS.register_module()
+class SegFormerHead(BaseDecodeHead):
+    """The MLP Head of segformer.
+
+    Args:
+        interpolate_mode: The interpolate mode of MLP head upsample operation.
+            Default: 'bilinear'.
+    """
+
+    def __init__(self, interpolate_mode='bilinear', **kwargs):
+        super(SegFormerHead, self).__init__(
+            input_transform='multiple_select', **kwargs)
+
+        self.interpolate_mode = interpolate_mode
+
+        num_inputs = len(self.in_channels)
+
+        self.convs = nn.ModuleList()
+        for i in range(num_inputs):
+            self.convs.append(
+                ConvModule(
+                    in_channels=self.in_channels[i],
+                    out_channels=self.channels,
+                    kernel_size=1,
+                    stride=1,
+                    norm_cfg=self.norm_cfg,
+                    act_cfg=self.act_cfg))
+
+        self.conv_fuse = ConvModule(
+            in_channels=self.channels * num_inputs,
+            out_channels=self.channels,
+            kernel_size=1,
+            norm_cfg=self.norm_cfg)
+
```

Comment:
Another suggestion is whether the kernel size of these two ConvModule can be passed in by Args.